Docker/requirements-api.txt

```diff
@@ -1,21 +1,21 @@
 astral==3.2
-timezonefinder==6.5.5
+timezonefinder==6.5.7
 pytz==2024.2
 zarr==2.18.3
-s3fs==2024.10.0
+s3fs
 ujson==5.10.0
 starlette==0.41.3
 uvicorn==0.32.1
 fastapi==0.115.5
 fastapi-utils==0.8.0
-boto3==1.35.68
+boto3==1.35.73
 scipy==1.14.1
 orjson==3.10.12
-psutil==6.1.0
+psutil==5.9.0
```

Comment:
When I setup the file I just used whatever was the latest version was at the time. I'm guessing version 6.1 didn't play nice with the setup?